Bash_scripts/HW13/script_hw13.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
I found out this one:
```
➜  ~ ./test.sh
Please enter a number A	3
Please enter a number B	Five
Please enter a number C	SIX
./test.sh: line 14: ${A,,}: bad substitution
./test.sh: line 15: ${B,,}: bad substitution
./test.sh: line 16: ${C,,}: bad substitution
A equal B equal C
➜  ~ ./test.sh
```

`Six` is out of range.
I think it will be great if you add validation for input variables.
It should be a number or it should be in range of 
```
declare -i one=1
 declare -i two=2
 declare -i three=3
 declare -i four=4
 declare -i five=5
 ```

Reply:
For `text` number please use array to pass throw it.